centos_ci.sh

```diff
@@ -224,6 +278,16 @@ else
     cd gopath/src/github.com/minishift/minishift
     make gen_adoc_tar IMAGE_UID=$(id -u)
     docs_tar_upload $PASS
+  elif [[ "$JOB_NAME" = "minishift-release" ]]; then
+    GIT_USER="minishift-bot"
+    prepare;
+    cd gopath/src/github.com/minishift/minishift
+    bump_to_release_version $GIT_USER;
+    make release
+    add_release_notes $GIT_USER;
+    # Generate docs tar and upload
+    make gen_adoc_tar IMAGE_UID=$(id -u)
+    docs_tar_upload $PASS
   else
     setup_kvm_docker_machine_driver;
```

Comment:
maybe everything in this branch could be in a function _build_and_test_. Maybe _run_tests_ can be renamed and used. After all it does more than just running test.